models/Byond/Endpoints/EndpointServerStatus.js

```diff
@@ -40,6 +40,9 @@ class EndpointASayMessage extends APIEndpoint {
         if (channel.id == config.discord_public_channel) {
           channel.sendEmbed(embed, embedmessage);
         }
+        if(channel.id == config.discord_botspam_channel) {
```

Comment:
Don't need to post it in both public and botspam, that just makes it annoying for subscribers.

Reply:
you do see how embedmessage isn't included in the one in botspam, right?

Reply:
like, that's the reason I added another if instead of just doing ||

Reply:
ah, my bad